composer.json

```diff
@@ -6,13 +6,13 @@
     "type": "project",
     "require": {
         "php": ">=7.1.3",
-        "laravel/lumen-framework": "5.7.*",
-        "vlucas/phpdotenv": "~2.2"
+        "laravel/lumen-framework": "5.8.*",
```

Comment:
Won't this break now because there isn't a 5.8 tag yet?

Reply:
No, as the travis build shows. :P

This is because of the following:

```json
    "minimum-stability": "dev",
    "prefer-stable": true
```

The same is true of `laravel/laravel`'s develop branch.